cmd/os/macos/InvariantDisks/.gitignore

```diff
@@ -0,0 +1,3 @@
+/DerivedData/
+*.xcworkspace/
+xcuserdata/
```

Comment:
is CMD the right location for InvariantDisks?
Maybe also add a short readme listing what this thing is actually for, for those of us not familiar with the OSX codestack?

Reply:
Made a cmd/os/macos/ so that other platforms wouldn't need to worry about it, but some general description could be nice (same goes for ZOL cmds I have no idea what they do)  As it happens, InvariantDisk is a separate process started by launchd to maintain a symlink-tree of storage devices, ie, /var/run/disk/by-serial/X  -> /dev/diskX. To work around disk renumbering problem.

Reply:
Same indeed also goes for some ZoL commands, yes...

With that description, as a sub process is cmd the right location?